sdk/ml/azure-ai-ml/azure/ai/ml/_internal/_schema/command.py

```diff
@@ -4,32 +4,46 @@
 from marshmallow import fields
 
 from azure.ai.ml._internal._schema.node import InternalBaseNodeSchema, NodeType
-from azure.ai.ml._schema import NestedField, StringTransformedEnum
-from azure.ai.ml._schema.component.retry_settings import RetrySettingsSchema
-from azure.ai.ml._schema.core.fields import DistributionField
+from azure.ai.ml._schema import (
+    NestedField,
+    UnionField,
+    RegistryStr,
+    AnonymousEnvironmentSchema,
+    ArmVersionedStr
+)
+from azure.ai.ml._schema.core.fields import DumpableEnumField
+from azure.ai.ml._schema.job import ParameterizedParallelSchema, ParameterizedCommandSchema
 from azure.ai.ml._schema.job.job_limits import CommandJobLimitsSchema
-from azure.ai.ml._schema.job_resource_configuration import JobResourceConfigurationSchema
+from azure.ai.ml.constants._common import AzureMLResourceType
 
 
-class CommandSchema(InternalBaseNodeSchema):
-    type = StringTransformedEnum(allowed_values=[NodeType.COMMAND], casing_transform=lambda x: x)
-    compute = fields.Str()
-    environment = fields.Str()
+class CommandSchema(InternalBaseNodeSchema, ParameterizedCommandSchema):
+    class Meta:
+        exclude = ["code", "distribution"]  # internal command doesn't have code & distribution
```

Comment:
why internal command does not have code?

Reply:
hmm... Code is a component level attribute. For v2 Command, its `code` is directly linked to `self.component.code`; For internal Command, I have checked v1.5 doc and code is just introduced as an attribute in component spec there; such a short-cut seems not necessary. 

PS: code is still in InternalComponentSchema and is shared by all internal components